src/proteus/proteus.py

```diff
@@ -229,12 +231,6 @@ def start(self, *, resume: bool = False):
             self.loops["init"] = self.loops["init_loops"] + 1
         log.info(" ")
 
-        # Create lockfile for keeping simulation running
-        self.lockfile = CreateLockFile(self.directories["output"])
```

Comment:
Not necessary anymore?

Reply:
This is still necessary, I've just moved it further up in the file. See response for download_sufficient_data.